site/topics/genetic-operators/genetic-operators.rst

```diff
@@ -7,6 +7,103 @@ Genetic Operators
 Genetic Operators for Binary Representations
 ============================================
 
+* Binary representations are those that only contain two values, which are typically ``0``\s and ``1``s
+
+
+Crossovers
+----------
+
+One Point Crossover
+^^^^^^^^^^^^^^^^^^^
+
+* Randomly select an index
+* Exchange the elements between the chromosomes after that index
+* This crossover is not particularly *destructive*
+
+    * The amount in which the information within the chromosomes gets changed is low
+
+
+* This crossover works well when element adjacency in the chromosome is important
+
+
+.. figure:: one_point_crossover.png
+    :width: 450 px
+    :align: center
+
+    Result of applying one point crossover on two chromosomes. This example has index ``5`` as the randomly selected
+    crossover point, thus, all elements after index ``5`` are exchanged between the parent chromosomes. Although this
+    example shows one parent containing only ``0``\s and the other containing only ``1``\s, this is not a requirement;
+    the parents could contain both ``0``\s and ``1``\s.
+
+
+:math:`n` Point Crossover
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Randomly select :math:`n` indices
+* Exchange the elements between every other pair of indices the indices
+
+    * If an odd number of indices, exchange the elements from the last index to the end
+
+
+* This is a generalization of one point crossover
+* :math:`n=2` is popular (two point crossover)
+
+.. figure:: n_point_crossover.png
+    :width: 450 px
+    :align: center
+
+    Result of applying :math:`n` point crossover where :math:`n=3`. The randomly selected indices in this example are
+    ``5``, ``8``, and ``14``. All elements between indices ``5`` and ``8`` (exclusively) are exchanged along with all
+    the elements from index ``14`` to the end of the chromosome.
+
+
+Uniform Crossover
+^^^^^^^^^^^^^^^^^
+
+* Select some random number of indices at random
+* Exchange the elements at those indices
+* Often implemented by giving each index a 50/50 chance to be selected for crossover
+
+* This crossover is relatively *destructive*
+* Not particularly effective on chromosomes where element adjacency is important
+
+.. figure:: uniform_crossover.png
+    :width: 450 px
+    :align: center
+
+    Result of applying uniform crossover where the a total of :math:`8` values were exchanged.
```

Comment:
and mixing math format with code -- at least stay consistent

Reply:
The goal is to try to use math-ie stuff for just numbers, and code-like style for things that are like, code. 

So, the idea is, it something is at index 8, it would be `8`, if the value at index `8` is 7, it would be math-ie 7.

That said, there is an issue with consistency throughout that goes back through all content.

Reply:
You're making busy work for yourself. Ain't no one going to be confused by 7 vs `7` vs _7_.